spec/migrations/20191112132759_copy_owner_id_to_group_id_for_dashboards_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_migration
+
+describe CopyOwnerIdToGroupIdForDashboards do
+  let(:miq_set)          { migration_stub(:MiqSet) }
+  let(:name)             { "Hello Dashboard" }
+  let(:group_id)         { 1 }
```

Comment:
Don't hardcode the group_id as this is invalid in a multi-region mode.  I believe there are specs where we need to create ids from an appropriate region, so search the specs for that...something around ActiveRecord::IdRegions.

Reply:
As agreed, we'll keep this as is, as it seems we don't have an existing mechanism for generating regionalized ids.